amplify/backend/function/questionslist/src/enums.js

```diff
@@ -1,34 +1,26 @@
 // Map of sheet values to enums in database
-const Difficulty = {
-  'Beginner': 'BEGINNER',
+const Levels = {
+  'Starter': 'STARTER',
   'Intermediate': 'INTERMEDIATE',
-  'Expert': 'ADVANCED',
+  'Guru': 'GURU',
 }
 
-const Cost = {
+const Costs = {
   '$': 'ONE',
   '$$': 'TWO',
   '$$$': 'THREE',
-  '$$$$': 'FOUR',
```

Comment:
While you're awaiting @shravanhariharan2's response I think the reasoning may have been because the figma omits `$$$$`. I could be wrong.

<img width="264" alt="Screen Shot 2021-04-11 at 5 06 25 PM" src="https://user-images.githubusercontent.com/37908451/114326157-41164200-9ae8-11eb-868d-5a1440ade15a.png">

Reply:
^ this, as well as the fact that `$$$$` label in the OG measure list was also removed

Reply:
got it, that makes sense. I didn't catch that on the list or on figma, so I just wanted to clarify. In that case, great work! I'll approve rn